std/experimental/ndslice/slice.d

```diff
@@ -383,6 +416,17 @@ pure nothrow @nogc unittest
     static assert(isRandomAccessRange!S);
 }
 
+///
+pure nothrow @nogc unittest
```

Comment:
The unittest exists to show how to use flags with slice tuple. Comment was added.